src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import core.basesyntax.animals.Animal;
+import core.basesyntax.animals.Penguin;
+import core.basesyntax.animals.Shark;
+import core.basesyntax.animals.Sparrow;
+import core.basesyntax.animals.Tiger;
+import core.basesyntax.buildings.Cage;
+import core.basesyntax.buildings.Zoo;
+import core.basesyntax.skills.Fly;
+import core.basesyntax.skills.Swim;
+
+public class Main {
+    public static void main(String[] args) {
+        Cage<Animal> cage = new Cage<>();
+        Zoo myZoo = new Zoo(cage);
+        Animal shark = new Shark("Bit");
+        cage.add(shark);
+        Animal penguin = new Penguin("Lucky");
```

Comment:
```suggestion
        Penguin penguin = new Penguin("Lucky");
        acquarium.add(penguin);
```

You can create a more specific cage for Pinguin. Like: Cage<? extends Birds, Swim>